tools/ci/illegal_dme_files.py

```diff
@@ -0,0 +1,43 @@
+# Naive search for illegal files. Has no semantic knowledge, just a lexical
+# search for specific endings in a .dme
+#
+# This is basically a slightly edited verison of unticked_files.py.
+# Look there for precise documentation on the methods used here.
+from pathlib import Path, PureWindowsPath
+import argparse
+import sys
+
+INCLUDED_FILES = [
+    'paradise.dme'
+]
+
+ILLEGAL_FILES = (
+    '.dmm'
+)
+
+def get_illegal_files(root: Path):
+    illegal_files = set()
+    for includer in INCLUDED_FILES:
+        with open(root / includer, 'r') as f:
+            # I've tried to optimize this, but this was the best I could get. Try placing the strips elsewhere if you dare
+            lines = [line for line in f.readlines() if line.rstrip('\r\n').strip('"').endswith(ILLEGAL_FILES)]
+            included = [line.replace('#include ', '').rstrip('\r\n').strip('"') for line in lines]
+            if(len(INCLUDED_FILES) > 1):
+                print(f'Found {len(included)} illegal files in {root / includer}')
+            illegal_files.update([root / Path(includer).parent / Path(PureWindowsPath(i)) for i in included])
+
+    return illegal_files
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("root", help="paracode root directory")
+    args = parser.parse_args()
+
+    # Windows quoting behavior for directories adds trailing double-quote
+    illegal_files = get_illegal_files(Path(args.root.strip('"')))
+    if illegal_files:
+        print(f'Found {len(illegal_files)} illegal files:')
```

Comment:
make sure the error makes it clear what the user has to do. e.g. "Found 3 files not meant to be included:"

also the actual output is a little confusing:
```
Found 1 illegal files in /home/runner/work/Paradise/Paradise/paradise.dme
Found 1 illegal files:
/home/runner/work/Paradise/Paradise/_maps/map_files/cyberiad/cyberiad.dmm
```

it would be nice if it were clearer that the illegal file found was in the listed DME file, e.g.

```
Found 1 illegal files in /home/runner/work/Paradise/Paradise/paradise.dme:
/home/runner/work/Paradise/Paradise/_maps/map_files/cyberiad/cyberiad.dmm

Found 2 illegal files in /home/runner/work/Paradise/Paradise/foo.dme:
/home/runner/work/Paradise/Paradise/_maps/map_files/cyberiad/cyberiad.dmm
/home/runner/work/Paradise/Paradise/_maps/map_files/cerestation/cerestation.dmm

```

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/37b1efe5-957f-41fb-b0a6-f0d49bdbe6c9)